mopidy/core/tracklist.py

```diff
@@ -398,35 +398,24 @@ def filter(self, criteria=None, **kwargs):
 
             # Returns tracks with TLIDs 1, 2, 3, or 4 (tracklist ID)
             filter({'tlid': [1, 2, 3, 4]})
-            filter(tlid=[1, 2, 3, 4])
-
-            # Returns track with IDs 1, 5, or 7
-            filter({'id': [1, 5, 7]})
-            filter(id=[1, 5, 7])
 
             # Returns track with URIs 'xyz' or 'abc'
             filter({'uri': ['xyz', 'abc']})
-            filter(uri=['xyz', 'abc'])
-
-            # Returns tracks with ID 1 and URI 'xyz'
-            filter({'id': [1], 'uri': ['xyz']})
-            filter(id=[1], uri=['xyz'])
-
-            # Returns track with a matching ID (1, 3 or 6) and a matching URI
-            # ('xyz' or 'abc')
-            filter({'id': [1, 3, 6], 'uri': ['xyz', 'abc']})
-            filter(id=[1, 3, 6], uri=['xyz', 'abc'])
 
         :param criteria: on or more criteria to match by
         :type criteria: dict, of (string, list) pairs
         :rtype: list of :class:`mopidy.models.TlTrack`
+
+        .. deprecated:: 1.1
+            Providing the criteria  via ``kwargs`` is no longer supported.
```

Comment:
s/  via/ via/ (two spaces)

Based on how the deprecated directive is used in other projects, I suggest removing the "is no longer supported" part.